docs/rules/no-restricted-modules.md

```diff
@@ -1,29 +1,35 @@
 # Disallow Node.js modules (no-restricted-modules)
 
-Disallowing usage of specific Node.js modules can be useful if you want to control the available methods, a developer can
-use, to implement a feature.
+A module in Node.js is a simple or complex functionality organized in single or multiple JavaScript files which can be reused throughout the Node.js
+application.The keyword `require` is used in Node.js/CommonJS to import modules into an application.This way you can have dynamic loading where the loaded module name isn't predefined /static, or where you conditionally load a module only if it's "truly required".
 
-This way you can block usage of the `fs` module if you want to disallow file system access.
-Blocking the `os` module can be useful if you don't want to allow any operating system specific code.
+Why you would want to restrict a module?
+
+Disallowing usage of specific Node.js modules can be useful if you want to control the available methods, a developer can use, to implement a feature.For Example,You can block usage of the `fs` module if you want to disallow file system access.
```

Comment:
Typos
 - Missing a space after the period: `to implement a feature.For Example,`
 - `Example` does not need to be capitalized

I think this first sentence would be clearer as:
`Disallowing usage of specific Node.js modules can be useful if you want to limit the available methods a developer can use.`